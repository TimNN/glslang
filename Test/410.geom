#version 410 core

void main()
{
    gl_ViewportIndex = 7;
}
