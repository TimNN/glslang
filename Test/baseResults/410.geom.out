Warning, version 410 is not yet complete; some version-specific features are present, but many are missing.
ERROR: 0:5: 'gl_ViewportIndex' : undeclared identifier 
ERROR: 1 compilation errors.  No code generated.

ERROR: node is still EOpNull!
0:3  Function Definition: main( (void)
0:3    Function Parameters: 
0:5    Sequence
0:5      move second child to first child (float)
0:5        'gl_ViewportIndex' (float)
0:5        Constant:
0:5          7.000000
0:?   Linker Objects

